### PR TITLE
Update river.json

### PR DESCRIPTION
**Pull request by moderator10000078**

### CODE REVIEW

File: bromine/pvp/maps/river.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		"ru": "Река"
 	},
 	"jokes": {
-		"en": "This book may contain your joke. Suggest via Discord.",
-		"ru": "В этой книге может быть твой анекдот. Предложи в Discord."
+		"en": "A blind man walks into a shop and starts twirling a chew toy dog over his head\n - Are you all right? - The salesman asks.\n - I'm just looking around.",
+		"ru": "Заходит слепой в магазин и начинает крутить над головой собаку-повадыря\n - У вас всё хорошо? - спросил продавец\n - Да так, осматриваюсь."
```
Comment (VidTu) [inline bromine/pvp/maps/river.json:11]: Newline fixes
```suggestion
		"ru": "Заходит слепой в магазин и начинает крутить над головой собаку-повадыря\n- У вас всё хорошо? - спросил продавец\n- Да так, осматриваюсь."
```

File: bromine/pvp/maps/river.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
 		"ru": "Река"
 	},
 	"jokes": {
-		"en": "This book may contain your joke. Suggest via Discord.",
-		"ru": "В этой книге может быть твой анекдот. Предложи в Discord."
+		"en": "A blind man walks into a shop and starts twirling a chew toy dog over his head\n - Are you all right? - The salesman asks.\n - I'm just looking around.",
```
Comment (VidTu) [inline bromine/pvp/maps/river.json:10]: Grammar fixes
```suggestion
		"en": "A blind man walks into a shop and starts twirling a chef-dog over his head\nThe salesman asks: \"Are you alright?\"\nThe man responds: \"Nevermind, I'm just looking around...\"",
```

**Review by VidTu — CHANGES_REQUESTED** [review]
:pig_nose:

File: bromine/pvp/maps/river.json (hunk 2 above)
Comment (VirtualFreeEx) [reply bromine/pvp/maps/river.json:10]:
```suggestion
		"en": "A blind man walks into a shop and starts twirling a chef-dog over his head\nThe salesman asks: \"Are you alright?\"\nThe man responds: \"Don't worry, I'm just looking around...\"",
```

**Review by VidTu — APPROVED** [review]

**Review by VirtualFreeEx — APPROVED** [review]
LGTM.

**Pull request merged**